pydantic_extra_types/pandas_types.py

```diff
@@ -0,0 +1,49 @@
+from typing import Any, List, Tuple, Type, TypeVar, Union
+
+from pydantic import GetCoreSchemaHandler
+from pydantic_core import core_schema
+
+try:
+    import pandas as pd
+except ModuleNotFoundError:  # pragma: no cover
+    raise RuntimeError(
+        '`Series` requires "pandas" to be installed. You can install it with "pip install pandas"'
+    )
+
+T = TypeVar('T', str, bytes, bool, int, float, complex, pd.Timestamp, pd.Timedelta, pd.Period)
+
+
+class Series:
+    def __init__(self, value: Any) -> None:
+        self.value = pd.Series(value)
+
+    @classmethod
+    def __get_pydantic_core_schema__(
+        cls, source: Type[Any], handler: GetCoreSchemaHandler
+    ) -> core_schema.AfterValidatorFunctionSchema:
+        return core_schema.general_after_validator_function(
+            cls._validate,
+            core_schema.any_schema(),
```

Comment:
I don't think this is the best we can do. Is it? A series should be an `array` of some type, and I guess we are able to get this type as well.